client/blocks/login/continue-as-user.jsx

```diff
@@ -50,41 +56,60 @@ function ContinueAsUser( { currentUser, redirectUrlFromQuery, onChangeAccount }
 	// like that, but it is better than the alternative, and in practice it should happen quicker than
 	// the user can notice.
 
+	const notYouText = isSignUpFlow
+		? translate( 'Not you?{{br/}} Sign out or log in with {{link}}another account{{/link}}', {
+				components: {
+					br: <br />,
+					link: (
```

Comment:
Should we declare this component separately? It seems to be duplicated across the 2 strings.

Reply:
Great catch!